digital-deck-server/model/cards.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
```

Comment:
Do we need this file? I don't think we need to store the texture of the cards in the DB, do we? I think for our purposes, we can just store this information in the frontend as a constant array of objects. Ie. 
```
const CARDS = [
    {
       id: 0,
       text: link to back of card texture
    },
    {
       id: 1,
       text: link to ace of hearts texture
    },
    {
        id: 2,
        text: link to 2 of hearts texture
    } etc.
];
```